Pod/Classes/Core/RNDynamicScreenDimensions.m

```diff
@@ -0,0 +1,112 @@
+#import "RNDynamicScreenDimensions.h"
+#import <UIKit/UIKit.h>
+#import <React/RCTComponent.h>
+#import <Emission/AREmission.h>
+#import <objc/runtime.h>
+
+@interface RNDynamicScreenDimensions ()
+@property (nonatomic, strong, readwrite) NSDictionary *currentDimensions;
+@property (nonatomic, readwrite) BOOL isBeingObserved;
+@end
+
+
+NSDictionary *getSafeAreaInsets()
```

Comment:
Marking these functions as `static` will allow the compiler to inline them. Unsure what the default is.